src/Components/Publishing/__stories__/ArticleStandard.story.tsx

```diff
@@ -1,7 +1,7 @@
 import { storiesOf } from "@storybook/react"
 import React from "react"
 import styled from "styled-components"
-import { Article } from "../Article"
+import { Article } from "Components/Publishing/Article"
```

Comment:
Thanks for adding support for this

Reply:
Ah, just remembered about the `roots` field in jest config: https://github.com/artsy/prediction/blob/master/package.json#L38

Reply:
can try that too